File: 01_Cpp/00_Co_ban/Cau_truc_dieu_khien/B17_Kiem_Tra_Chu_Cai.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;

int main(){
	char a; cin>>a;
	if(a >= 'A' && a <= 'Z'){
	    cout<<"UPPER";
	}
	else if(a >= 'a' && a <= 'z'){
	    cout<<"LOWER";
	}
	else if(a<='9' && a>='0'){
	    cout<<"DIGIT";
	}
	else{
	    cout<<"SPECIAL";
	}	
	return 0;
}

File: Java/JavaSQL/src/Bai_Tap/Chapter_02_Cau_Truc_Re_Nhanh/B17_Kiem_Tra_Chu_Cai.java
package Bai_Tap.Chapter_02_Cau_Truc_Re_Nhanh;

import java.util.Scanner;

public class B17_Kiem_Tra_Chu_Cai {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        char c = sc.nextLine().charAt(0);

        if (Character.isUpperCase(c)) {
            System.out.println("UPPER");
        } else if (Character.isLowerCase(c)) {
            System.out.println("LOWER");
        } else if (Character.isDigit(c)) {
            System.out.println("DIGIT");
        } else {
            System.out.println("SPECIAL");
        }
    }
}
